satellite/metainfo/metainfo.go

```diff
@@ -836,9 +862,15 @@ func convertProtoToBucket(req *pb.BucketCreateRequest, projectID uuid.UUID) (sto
 
 func convertBucketToProto(ctx context.Context, bucket storj.Bucket) (pbBucket *pb.Bucket) {
 	rs := bucket.DefaultRedundancyScheme
```

Comment:
**Rational for code Changes:**
added necessary changes to handle conversion from/to proto and storj.bucket